buildscripts/azure/azure-linux-macos.yml

```diff
@@ -13,8 +13,6 @@ jobs:
 
   steps:
     - script: |
-        if [ "$(uname)" == "Linux" ] && [[ "$CONDA_SUBDIR" == "linux-32" || "$BITS32" == "yes" ]]; then sudo apt-get install -y libc6-dev-i386; fi
-        if [ "$(uname)" == "Linux" ] && [[ "$CONDA_SUBDIR" != "linux-32" && "$BITS32" != "yes" ]]; then sudo apt-get install -y gdb; fi
```

Comment:
Comment only: line 17 here is not related to this PR, but is from the already-merged #8422.